v20.2/alter-column.md

```diff
@@ -59,6 +59,8 @@ For an example of `ALTER COLUMN TYPE`, see [Increase a column type's precision](
 
 ## Examples
 
```

Comment:
On this page, all examples must be within a subsection of `## Examples`. To make a subsection, create a title with a level-3 header (e.g., `### Update the data type of two columns`). 

For reference, see the example directly following `## Examples`. It has the level-3 header `### Set or change a DEFAULT value`.